client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -279,6 +406,10 @@ function mediaButton( editor ) {
 			// autosave take effect
 			editor.fire( 'change' );
 		}
+
+		if ( numOfImagesToUpdate === 0 ) {
+			MediaActions.edit(  selectedSite.ID, { ...dirtyImage, isDirty: false } );
```

Comment:
If there are no more images to update, let's set the image `.isDirty` prop to false so the image is not updated each time other images are updated.